client/state/themes/reducer.js

```diff
@@ -301,7 +303,15 @@ export const queries = ( () => {
 		};
 	}
 
-	// Time after which queries storred in IndexedDb will be invalidated.
+	function fromApi( theme ) {
+		if ( ! theme || ! theme.description ) {
+			return theme;
+		}
+
+		return Object.assign( {}, theme, { description: decodeEntities( theme.description ) } );
```

Comment:
Could we do `{ ...theme, description: decodeEntities( theme.description ) }` here instead? Is there a reason we're using `Object.assign` here in particular?

Reply:
Sounds good - I'll change that.